content/en/docs/glossary/_index.md

```diff
@@ -87,8 +93,22 @@ SSO
 Theme
 : A set of templates, styles, and code that define the presentation layer of a site. See [Theming Drupal](https://www.drupal.org/docs/theming-drupal).
 
+YMCA Canada
+Y Canada
+: The Canadian YMCA Federation is made up of a national office and Member Associations across the country. See [ymca.ca](https://www.ymca.ca/).
+
+YMCA North American Network
+YNAN
+: The organization representing the largest YMCA associations in the country.
+
+YMCA of the North
 Y North
-: Formerly YMCA of the Twin Cities. One of the founding associations of Open Y. Lovers of llamas.
+: Formerly _YMCA of the Twin Cities_. One of the founding associations of Open Y. Lovers of llamas. See [ynorth.org](https://www.ymcanorth.org/).
+
+YMCA of the USA
+YUSA
+Y-USA
+: The central office for YMCAs in the USA. Headquartered in Chicago. [YMCA of the USA](https://ymca.org/).
```

Comment:
Instead of "central", can you instead say "national office", as central implies a hierarchy or top-down organization structure. If the words "support organization" can be worked in, that may help also.

Reply:
@TomScarboroughYMCA how's this?

```suggestion
: The national resource office, supporting thousands of YMCAs, tens of thousands of staff, and hundreds of thousands of volunteers in 10,000 communities across the USA. Headquartered in Chicago. [YMCA of the USA](https://ymca.org/).
```